Assets/_Laz/Scripts/Environment/ElectricGateBehaviour.cs

```diff
@@ -17,10 +17,13 @@ public class ElectricGateBehaviour : MonoBehaviour
         [SerializeField, Range(0f, 50f), Tooltip("Proxy for gate's z scale. Adjust this to increase the width of the gate")] 
         private float _gateWidth = 10f;
         [SerializeField] 
-        private Transform _leftPost, _rightPost, _gateTransform = default;
+        private Transform _leftPost, _rightPost = default;
 #endif
+        [SerializeField] private Transform _gateTransform = default;
         [SerializeField] private Renderer _gateRenderer = null;
 
+        [SerializeField, Tooltip("Initial state for the gate, if its set up with a puzzle. If true, gate will start as on and puzzles will deactivate gate.")] private bool _initialState = true;
```

Comment:
can you move this `private bool`
to the next line so it's easier to read

Reply:
You can probably pass the _initialState inside `ElectricGate` and have it handle the logic in side there.